client/src/components/Dataset/DatasetList.vue

```diff
@@ -100,16 +100,13 @@ async function load(concat = false) {
     }
 }
 
-async function onCopyDataset(item: HDASummary) {
+async function onCopyContent(item: HDASummary) {
```

Comment:
The type annotation here isn't correct if you're calling this function `onCopyContent`